events/test_routes.py

```diff
@@ -59,6 +60,13 @@ def test_add_invalid_event(self):
 
         self.assertEqual(self.coll.count_documents({}), 0)
 
+    def test_db_not_defined(self):
```

Comment:
Docstring? It could help to run `pylint -v app` and fix any style errors before requesting a PR.

Reply:
Added 6775c69